### ISSUE TITLE
A predictable URL for latest build would be helpful 

### ISSUE THREAD

**Issue by qrkourier** [openziti/ziti#472]
GitHub provides a "latest" downloads shortcut for release artifacts, but using this with a predictable URL depends upon the releaser using a predictable artifact name. This repo "ziti" uses a version-specific artifact name, and so it's not possible to predict the URL to download the latest build.

For example, this is a valid, albeit unpredictable, URL for the latest x86_64 build for Linux: https://github.com/openziti/ziti/releases/latest/download/ziti-linux-amd64-0.22.0.tar.gz

I'm unable to create a CI job that consumes this unpredictable URL because the artifact name is only valid for the current release.

A predictable latest artifact URL would be like this: https://github.com/openziti/ziti/releases/latest/download/ziti-linux-amd64.tar.gz
and that predictable URL would today lead to the same artifact as this fictitious, release-specific URL:
https://github.com/openziti/ziti/releases/download/v0.22.0/ziti-linux-amd64.tar.gz

**Comment by dovholuknf** [comment]
can't you interrogate the github api and look under 'assets' for the one you want? i feel like i must be missing something? find the asset that the name 'starts with' "ziti-linux-amd64" and use the download url?

https://api.github.com/repos/openziti/ziti/releases/latest

```
      "browser_download_url": "https://github.com/openziti/ziti/releases/download/v0.22.0/ziti-linux-amd64-0.22.0.tar.gz"
```

**Comment by qrkourier** [comment]
Yes, but that's why GitHub provides the "latest" shortcut, to obviate all of us writing that same logic that solves an easily-anticipated problem.
